Assignment 1/Exercise3.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Exercise3
+ */
+import java.io.Console;
+
+public class Exercise3 {
+
+    public static void main(String[] args) {
+        Console console= System.console();
+        String word = console.readLine("Enter a palindrom: ");
+        int start = 0;
+        int end = word.length()-1;
+        Boolean palin = true;
+        while (start <= end)
+          {
+              if (word.charAt(start) != word.charAt(end))
+                palin = false;
```

Comment:
Awesome!, the only think I would like to suggest here is that after `palin = false` you can also do a `break;` this will force the while to stop, which is something desired since you already found that the `palin` is false so there is no reason to continue checking :)

Note: The `if` will need brackets `{}` since now you will have two lines to execute for that condition